api/src/test/java/org/apache/iceberg/expressions/TestInclusiveManifestEvaluator.java

```diff
@@ -347,4 +349,52 @@ public void testStringStartsWith() {
     shouldRead = ManifestEvaluator.forRowFilter(startsWith("some_nulls", "1"), SPEC, false).eval(FILE);
     Assert.assertFalse("Should skip: range doesn't match", shouldRead);
   }
+
+  @Test
+  public void testIntegerIn() {
```

Comment:
I think this also needs tests for all the null cases. Can you add tests for the `all_nulls` and `no_nulls` columns to all of the suites?

Reply:
Done.